Tests/StoreKitUnitTests/PurchasesOrchestratorSK2Tests.swift

```diff
@@ -424,31 +323,24 @@ class PurchasesOrchestratorSK2Tests: PurchasesOrchestratorTestCase {
         mockListener.mockTransaction = .init(try await self.simulateAnyPurchase())
 
         let product = try await self.fetchSk2Product()
-
-        let package = Package(identifier: "package",
-                              packageType: .monthly,
-                              storeProduct: StoreProduct(sk2Product: product),
-                              offeringIdentifier: "offering")
-
-        _ = try await self.orchestrator.purchase(sk2Product: product, package: package, promotionalOffer: nil)
```

Comment:
I've removed the package parameter from a bunch of tests because it's already tested in `testPurchasePostsReceipt` and makes tests easier to read.